### PR TITLE
Linda

### PR DESCRIPTION
**Pull request by LindaNayeli104**

### CODE REVIEW

**Comment by OscarFM014** [comment]
Change the PR name

File: nbproject/private/private.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
 compile.on.save=true
-user.properties.file=/Users/oscarfernandez/Library/Application Support/NetBeans/13/build.properties
+user.properties.file=C:\\Users\\linda\\AppData\\Roaming\\NetBeans\\13\\build.properties
```
Comment (OscarFM014) [inline nbproject/private/private.properties:2]: you need to change this?

Comment (LindaNayeli104) [reply nbproject/private/private.properties:2]: NetBeans did it automatically.

Comment (OscarFM014) [reply nbproject/private/private.properties:2]: Ok, we should add a .gitignore related to Java projects in Netbeans

**Comment by OscarFM014** [comment]
LGTM

File: nbproject/private/private.properties (hunk 1 above)
Comment (LindaNayeli104) [reply nbproject/private/private.properties:2]: Good idea

**Pull request merged**